packages/headless/src/features/commerce/facets/facet-set/facet-set-reducer-helpers.ts

```diff
@@ -0,0 +1,19 @@
+import {CommerceFacetSetState} from './facet-set-state';
+import {CategoryFacetValueRequest} from './interfaces/request';
+
+export function handleCategoryFacetNestedNumberOfValuesUpdate(
```

Comment:
This is largely duplicated from the function of the same name in category-facet-set-reducer-helpers.ts

Duplicating the code can hardly be avoided in this case because the state is different, and "values" is named "currentValues" in the non-commerce category facet state requests.